src/eve-server/system/Container.cpp

```diff
@@ -83,6 +83,8 @@ CargoContainerRef CargoContainer::Spawn( ItemData &data) {
     containerRef->SetAttribute(AttrMass,          containerRef->type().mass(), false);          // Mass
     containerRef->SetAttribute(AttrVolume,        containerRef->GetPackagedVolume(), false);        // Volume
     containerRef->SetAttribute(AttrCapacity,      containerRef->type().capacity(), false);      // Capacity
+    containerRef->SetAttribute(AttrAnchoringDelay,60000, false);  // static 60 sec timer for now
```

Comment:
I think we should remove this since we shouldn't really be setting static attributes like this but instead using those from dogma in general. One way to get around this in your testing environment is making a db change so that the anchoring delays are shorter (if you wish). [An example of this would be the db migration here](https://github.com/idolord/evemu_Crucible/blob/staging/sql/migrations/20211011182454-ihub.sql)

Reply:
This isn't working, added entry in dB first then tried to set this attribute here but still containers instantily anchore and then cannot be unanchored still need work. An some figuring out m'y part.

Reply:
hoooooo delay attributes are dgmTypeAttributes not entityattribute ... that might explain why it isn't working ... me dumb.

Reply:
as this attribute doesn't exist it wil mostly be : 

> INSERT INTO dgmTypeAttributes (valueFloat, attributeID, typeID) VALUES(60000, 556, 3466);
> INSERT INTO dgmTypeAttributes (valueFloat, attributeID, typeID) VALUES(60000, 676, 3466);

but after testing it seems that, that alone doesn't fix the problem i might be missing something, will investigate ...